### DISCUSSION TITLE
RX-V477 confirmed working

### DISCUSSION THREAD

**Discussion by ChristophCaina** [mvdwetering/yamaha_ynca#144] [General]
Sure, it is an older device, but the YAMAHA RX-V477 seems to work flawlessly with this integration.
![image](https://user-images.githubusercontent.com/26391061/223062844-05943fc4-58d7-4aa2-8e57-89bde793f00b.png)

There's not much different compared to the default yamaha configuration in HomeAssistant, which you have to configure manually...

**Comment by mvdwetering** [comment]
Thanks for the feedback 👍 
I added the RX-V477 to the list in the README
